qaframework-bdd-tests/src/test/resources/features/refapp-3.x/01-login/login.feature

```diff
@@ -7,6 +7,6 @@ Feature: User Login
 
     Examples:
       | username   | password   | location          | ability |
-      | admin2     | Admin123   | Registration Desk | able    |
-      | wrong user | Admin123   | Registration Desk | unable  |
-      | admin2     | wrong pass | Registration Desk | unable  |
+      | admin     | Admin123   | Outpatient Clinic| able    |
+      | wrong user | Admin123   | Outpatient Clinic| unable  |
```

Comment:
@jayasanka-sack @pasindur99
could you be some information as to why the **Example 2** is failing 

I think usernames are entered before actual recognition of labels takes place

cc @kdaud @sherrif10

Reply:
Hi @jnsereko, 
Thanks for the PR.

Currently, these tests are failing due to many reasons.
ex:
- The UI takes a significant amount of time to load (because of the offline mode). Sometimes even it fails to load.
- It is running on a deployed server. Hence, data mutations affect existing tests.

We might need structural changes to address those problems. Hoping we can come up with proper solutions with the 3.x E2E GSoC project.

Reply:
> * The UI takes a significant amount of time to load (because of the offline mode). Sometimes even it fails to load.

I realised later that this was the actual problem @jayasanka-sack 
On `Example 2` loading of resources takes somewhat long and hence timeout error.